src/Parsers/ASTSetQuery.h

```diff
@@ -13,6 +13,7 @@ class ASTSetQuery : public IAST
 {
 public:
     bool is_standalone = true; /// If false, this AST is a part of another query, such as SELECT.
+    bool is_clone = false; /// If true, this AST is a clone from other part of the query and should not be printed in format()
```

Comment:
Minor: `printed in format()` --> `printed in FORMAT` (or did you mean `formatImpl()`?)

Reply:
I think it was `formatAST`. I'll improve the comment